src/object.d

```diff
@@ -1987,15 +1987,15 @@ auto byKey(T : V[K], K, V)(T* aa) pure nothrow @nogc
     return (*aa).byKey();
 }
 
-auto byValue(T : V[K], K, V)(T aa) pure nothrow @nogc
+auto byValue(T : V[K], K, V)(T aa) pure nothrow @nogc @trusted
 {
     import core.internal.traits : substInout;
 
     static struct Result
     {
         AARange r;
 
-    pure nothrow @nogc:
+    pure nothrow @nogc @trusted:
```

Comment:
To be fair, this is also applies to `std.range.front` for normal arrays when compiling in release mode without bounds checking, and that's marked as safe.